electronics/src/Config.h

```diff
@@ -0,0 +1,103 @@
+#ifndef CONFIG_H
+#define CONFIG_H
+
+#include "Handles.h"
+#include <PinNames.h>
+
+/**
+ *
+ * -----------------------------------------------
+ * SET MODE [Type of controller to be deployed to]
+ * -----------------------------------------------
+ * 
+ */
+
+// #define RAM // Red Alliance controller mode
+// #define BAM // Blue Alliance controller mode
+// #define SGM // Shield Generator controller mode
+#define STM // Scoring Table controller mode
+
+#ifdef RAM
+#define MODE 0
+#endif
+
+#ifdef BAM
+#define MODE 1
+#endif
+
+#ifdef SGM
+#define MODE 2
+#endif
+
+#ifdef STM
+#define MODE 3
+#endif
+
+#define global static const // Constant experssion accessable to all (cannot be modified)
```

Comment:
Giving this a define is an odd decision - it hides the meaning that "static const" gives that C++ programmers are used to.

Reply:
Yea, not very needed. I'll switch it back, this was for something fancy when global was a shortcut for a very long string of C++ keywords, but it ended up being only the two.